src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -207,9 +211,26 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!-- Reads the version of the compiler APIs that are currently being used in order to pick the correct Roslyn components. -->
+  <Target Name="_ResolveCompilerVersion"
```

Comment:
I'd like to see the Roslyn targets updated to set this version number soon so that this target is normally never actually run.

Reply:
@chsienki - any chance you can take care of this? I can open an issue in dotnet/roslyn if you'd like.

Reply:
We have an issue here https://github.com/dotnet/roslyn/issues/52265 

It's easy enough to ship in the .NET 6 timeframe by just hard coding it; my concern has always been around the maintainability and ensuring it never gets out of sync, but we can figure that out after the fact if we really need it here.